.pre-commit-config.yaml

```diff
@@ -11,7 +11,34 @@ repos:
     rev: 'v0.942'
     hooks:
     -   id: mypy
-        # stages: [push]
+        pass_filenames: false  # suppress the normal filename passing
+        files: ^freqtrade/  # filter the files down to a specific subdirectory
+        # pre-commit only supports running at the root of a repo since that's where
+        # git hooks run.  but it also allows running arbitrary code so you can
+        # step outside of those bounds
+        # note that `bash` will reduce your portability slightly
+        verbose: true
+        entry: mypy freqtrade
+        additional_dependencies: 
+        -    types-cachetools==5.0.0
+        -    types-filelock==3.2.5
+        -    types-python-dateutil==2.8.10
+        -    types-requests==2.27.15
+        -    types-tabulate==0.8.6
+        -    types-urllib3==1.26.11
```

Comment:
These dependencies will not update automatically via dependabot, which means they'll become outdated within weeks.

If this is the requirement - then we'll be removing mypy from the pre-commit hook again.